android/build.gradle

```diff
@@ -30,4 +33,29 @@ android {
     lintOptions {
         disable 'InvalidPackage'
     }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_7
+        targetCompatibility JavaVersion.VERSION_1_7
```

Comment:
lets make these VERSION_1_8 I believe that is best practices and allows some nice language features

Reply:
Agreed this was debugging code that I let slip through